cooked-validators/tests/Cooked/MockChain/Monad/StagedSpec.hs

```diff
@@ -72,66 +61,6 @@ tests =
                 validateTxSkel $ f $ txSkel [paysPK (walletPKHash $ wallet 2) (Pl.lovelaceValueOf 4200000)]
                 validateTxSkel $ g $ txSkel [paysPK (walletPKHash $ wallet 3) (Pl.lovelaceValueOf 4200000)]
                 validateTxSkel $ h $ txSkel [paysPK (walletPKHash $ wallet 4) (Pl.lovelaceValueOf 4200000)]
-           in interpret (somewhere f (tr id id id)) `imcEq` interpret (tr id (fromJust . f) id)
-      ],
-    testCase
-      "Somewhere is exponential in branch number"
```

Comment:
Maybe we should keep this test and the next test (*); I didn't see any test involving nested `modify`s and these were regression tests that uncovered bugs on previous implementations.

Reply:
Oh, I forgot that one (but it works). I'll add a test.